tests/composer.json.travis

```diff
@@ -0,0 +1,170 @@
+{
+  "name": "PrestaShop/PrestaShop",
+  "description": "PrestaShop offers a free, fully scalable, Open Source e-commerce solution.",
+  "type": "project",
+  "repositories": [
+    {"type": "vcs", "url": "https://github.com/prestashop/blockcart"},
```

Comment:
Please, only use `composer` to download the modules so the tests will truly reflect the real package